test/specs/anInitialSuite.e2e.ts

```diff
@@ -84,7 +84,7 @@ describe('An Initial Suite', async () => {
   });
 
   step('Set up the testing environment', async () => {
-    testSetup = new TestSetup('AnInitialSuite', false);
+    testSetup = new TestSetup('AnInitialSuite');
```

Comment:
``` suggestion
    testSetup = new TestSetup('CliCommandTest');
```

Reply:
This is AnInitialSuite hehe, I changed the name in the clicommands.test.ts